docs/reference/changelog.md

```diff
@@ -4,6 +4,97 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Darklang Release 7 (November 1, 2022)
+
+## Editor improvements
+
+- Remove expandable sidebar, make collapsed sidebar more reliable
+  ([#4511](https://github.com/darklang/dark/pull/4511))
+
+  TODO: choose a screenshot before/after pair that represents the visual changes
+  well
+
```

Comment:
```suggestion
```
I'd only included this TODO because I felt it'd be nice to include at least one visual per changelog.
In reflection, though, the visual changes here feel a bit too subtle to include?